src/Core/Settings.h

```diff
@@ -1215,7 +1215,12 @@ class IColumn;
     M(Bool, precise_float_parsing, false, "Prefer more precise (but slower) float parsing algorithm", 0) \
     M(DateTimeOverflowBehavior, date_time_overflow_behavior, "ignore", "Overflow mode for Date, Date32, DateTime, DateTime64 types. Possible values: 'ignore', 'throw', 'saturate'.", 0) \
     M(Bool, validate_experimental_and_suspicious_types_inside_nested_types, true, "Validate usage of experimental and suspicious types inside nested types like Array/Map/Tuple", 0) \
-
+    \
+    M(String, diagram_output_format_diagram_type, "SCATTER", "The type of diagram", 0)\
```

Comment:
s/`diagram_output_format`/`output_format_diagram` would be more consistent with other setting names, e.g. `output_format_parquet_row_group_size`.